az-flow-trigger-dependency-plugin/src/main/java/flowtrigger/DependencyCheck.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package azkaban.flowtrigger;
+package flowtrigger;
```

Comment:
I think all other azkaban package names start with `azkaban.`. Should we change it for this package? Is there an easy way to make this also `azkaban.flowtrigger` without losing the benefits of this PR? It would also make it easier for a trigger dependency developer to identify through which imports they depend on azkaban.

Reply:
good point, thanks